wsmaster/che-core-api-account/src/test/java/org/eclipse/che/account/spi/AccountValidatorTest.java

```diff
@@ -50,7 +57,22 @@ public void testNormalizeAccountName(String input, String expected) throws Excep
     public void testNormalizeAccountNameWhenInputDoesNotContainAnyValidCharacter() throws Exception {
         doThrow(NotFoundException.class).when(accountManager).getByName(anyString());
 
-        Assert.assertTrue(accountValidator.normalizeAccountName("#", "name").startsWith("name"));
+        assertTrue(accountValidator.normalizeAccountName("#", "name").startsWith("name"));
+    }
+
+    @Test
+    public void shouldChangeLast4CharactersInAccountNameIfAccountNameContains39CharactersAndAccountAlreadyExists() throws Exception {
```

Comment:
test name says `..AndAccountAlreadyExists` but I see another logic with `AndAccountDoesNotExist`. If I'm right please fix it and add new test to cover both of them.